### DISCUSSION TITLE
可否考虑随机ip的时候多点增加不同国家的优选ip节点?

### DISCUSSION THREAD

**Discussion by Zding89** [vvbbnn00/WARP-Clash-API#110] [Q&A]
我用作者你社区布置的，一般荷兰一半美国，我部署的全是美国，能多几个国家的随机吗，那样说不得就容易点遇上能过gpt的节点

**Comment by vvbbnn00** [comment]
优选IP节点的IP段目前似乎只有荷兰和美国，若您有获取其他接入点IP相关的信息，欢迎提供
关于连接ChatGPT的问题，与项目本身无关，请参考：https://github.com/vvbbnn00/WARP-Clash-API/issues/65
